source/index.html.md

```diff
@@ -246,13 +247,12 @@ curl \
       "affiliate_program_id": "adwords",
```

Comment:
Good call on removing the accepted terms.  I don't recall if there was a reason why "affiliate_details" is an array - that seems odd.

Reply:
yeah wasn't sure why it was an array. I was hesitant to even put affiliate details and comms preferences in here.

Reply:
I just changed the example to use an object (as defined in the request parameters)